libs/langchain/langchain/chains/graph_qa/prompts.py

```diff
@@ -78,9 +78,9 @@
 
 Generate the Kùzu dialect of Cypher with the following rules in mind:
 
-1. Do not use a `WHERE EXISTS` clause to check the existence of a property.
-2. Do not omit the relationship pattern. Always use `()-[]->()` instead of `()->()`.
-3. Do not include any notes or comments even if the statement does not produce the expected result.
+1. Do not omit the relationship pattern. Always use `()-[]->()` instead of `()->()`.
```

Comment:
We try to avoid changing prompts like this as it can silently change behavior for users. What are your thoughts?

Reply:
I think it's fine in this case (though I total get your point, changing prompts that may affect larger behavior isn't ideal by any means). However, here, I think the earlier prompt isn't suitable and I've tested the behaviour subjectively using multiple proprietary and open source LLMs, and it seems to not change the behaviour at all. Additionally, we have a rather small user base that's actively using LangChain right now, but this may grow in the coming months, so the sooner we get this in from a Kùzu perspective, the better 🙂.

Hope that makes sense!